cmd/podman/load.go

```diff
@@ -64,10 +65,10 @@ func loadCmd(c *cliconfig.LoadValues) error {
 	if runtime.Remote && len(input) == 0 {
 		return errors.New("the remote client requires you to load via -i and a tarball")
 	}
-	if len(input) == 0 {
-		input = "/dev/stdin"
-		c.Input = input
-
+	if input == "" {
```

Comment:
So, the reason I did `len(input) == 0` is because of that usage three lines up. For the sake of consistency, and minimizing maintenance programmer confusion (like, someone some day wondering "why is it this way in one place and the other way in the other?"), could you consistentize those two?

Actually, I think it might be even more readable as
```go
    if (whichever way you choose to check if string is empty) {
        if runtime.Remote {
            return errors.New()
        }
        ...
```

Reply:
I actually do not know why the runtime.Remote check is here. We should be able to support this on a remote system?

Reply:
Removing check.
@baude @jwhonce WDYT?